packages/tools/sandbox/src/components/renderingZone.tsx

```diff
@@ -69,7 +69,7 @@ export class RenderingZone extends React.Component<IRenderingZoneProps> {
                         "indirect-first-instance",
                     ],
                 },
-                antialiasing: antialias,
+                antialias,

```

Comment:
My bad, I forgot this change when I renamed `antialiasing` to `antialias` to align with WebGL!

Could you also perform this change in these files:
* tools/testTools/src/utils.ts
* tools/testTools/src/visualizationUtils.ts

Thanks!